easybuild/easyconfigs/b/Boost/Boost-1.55.0-goolf-1.7.20-Python-2.7.9.eb

```diff
@@ -0,0 +1,27 @@
+name = 'Boost'
+version = '1.55.0'
+
+homepage = 'http://www.boost.org/'
+description = """Boost provides free peer-reviewed portable C++ source libraries."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
+source_urls = [SOURCEFORGE_SOURCE]
+
+pythonversion = '2.7.9'
```

Comment:
I made this change, which required a Python 2.7.10 EB for goolf, so I've added that now too.